specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2022-01-01/redisenterprise.json

```diff
@@ -1763,6 +1805,133 @@
         "ids"
       ],
       "description": "Parameters for a Redis Enterprise Active Geo Replication Force Unlink operation."
+    },
+    "RegionSkuDetail": {
+      "title": "Region Sku Detail",
+      "description": "Details about the location requested and the available skus in the location",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "type": "string"
+        },
+        "locationInfo": {
+          "$ref": "#/definitions/LocationInfo",
+          "description": "Details about location and its capabilities"
+        },
+        "skuDetails": {
+          "$ref": "#/definitions/SkuDetail",
+          "description": "Details about available skus"
+        }
+      }
+    },
+    "RegionSkuDetails": {
+      "title": "Region Sku Details",
+      "description": "List of Region SKU Detail",
+      "type": "object",
+      "properties": {
+        "value": {
```

Comment:
Including 'nextLink' will make this api slightly more future proof in case we want to return paged results in future.